build.ps1

```diff
@@ -125,6 +125,7 @@ param(
     [switch]
     $IncludeKnownIssues,
 
+    [Parameter(Mandatory, ParameterSetName="SkipPesterInstall")]
```

Comment:
This definitely should not be mandatory...

A switch parameter should not be mandatory...

we should change this to `$ForcePesterInstall` and change the logic in the if to have an `-or $ForcePesterInstall.IsPresent`

Reply:
oh ok

Reply:
But I also feel like this should be opted in by default for the one-command builds concept

Reply:
It kind of is... if you don't have pester, it will install it... I have a function to detect the version and install, if it's older... do you want me to get the code?

Reply:
That really shouldn't be needed